rst_files/testing.rst

```diff
@@ -588,7 +592,7 @@ Lastly, we can trigger builds manually (without a new commit or PR) from the Tra
 .. figure:: /_static/figures/travis-trigger.png
     :scale: 60%
 
-To commit *without* triggering a build, simply add [ci skip] somewhere inside the commit message
+To commit *without* triggering a build, simply add "[ci skip]" somewhere inside the commit message
```

Comment:
Wasn't sure if we should code-highlight ``[ci skip]``

Reply:
Sure, looks good.